master/internal/spec_util_intg_test.go

```diff
@@ -0,0 +1,133 @@
+//go:build integration
+// +build integration
+
+package internal
+
+import (
+	"testing"
+
+	"github.com/determined-ai/determined/master/internal/config"
+	"github.com/determined-ai/determined/master/internal/mocks"
+	"github.com/determined-ai/determined/master/internal/sproto"
+	"github.com/determined-ai/determined/master/pkg/archive"
+	"github.com/determined-ai/determined/master/pkg/model"
+	"github.com/determined-ai/determined/master/pkg/tasks"
+	"github.com/determined-ai/determined/proto/pkg/utilv1"
+	"github.com/stretchr/testify/require"
+	k8sV1 "k8s.io/api/core/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func getMockResourceManager(poolName string) *mocks.ResourceManager {
```

Comment:
i added something like `MocKRM` in internal recently as a mock rm that basically always returns "OK". feel free to see if that is helpful. if you need specific functions mocked for your test to make sense, i think it's a lot more readable to mock them directly in your test, anyway.

Reply:
I think the way it is is good for now